packages/asset-swapper/src/utils/market_operation_utils/sampler_operations.ts

```diff
@@ -273,19 +297,33 @@ export const samplerOperations = {
             },
         };
     },
-    getMedianSellRate(
+    getBalancerSellQuotes(pool: BalancerPool, takerFillAmounts: BigNumber[]): SourceQuoteOperation<BalancerFillData> {
+        return {
+            source: ERC20BridgeSource.Balancer,
+            fillData: { poolAddress: pool.id },
+            ...samplerOperations.constant(takerFillAmounts.map(_.curry(computeBalancerSellQuote)(pool))),
```

Comment:
why not drop the lodash and just `map(amt => computeBalancerSellQuote(pool, amt))`?